src/presence_analyzer/views.py

```diff
@@ -20,13 +20,30 @@
 
 log = logging.getLogger(__name__)  # pylint: disable=invalid-name
 
+links = {
+    'Presence by weekday': '/presence_weekday.html',
+    'Presence mean time': '/mean_time_weekday.html',
+    'Presence start-end': '/presence_start_end.html',
+}
+
 
 @app.route('/')
 def mainpage():
     """
     Redirects to front page.
     """
-    return redirect('/static/presence_weekday.html')
+    return render_template('presence_weekday.html', pages=links)
+
+
+@app.route('/<template>')
+def dynamic_routes(template):
+    """
+    Create dynamic routes and render template.
+    """
+    try:
+        return render_template(template, pages=links)
+    except Exception:
```

Comment:
U shouldn't catch generic exception. find out what exception is thrown when there's no such template.